public/request/achievement/update-base.php

```diff
@@ -14,7 +15,7 @@
     'points' => 'required|integer',
 ]);
 
-$achievement = GetAchievementData((int) $input['achievement']);
+$achievement = Achievement::find($input['achievement']);
```

Comment:
Attempting to update achievement data from achievement page generates `Indirect modification of overloaded element of LegacyApp\Platform\Models\Achievement has no effect` error.

Reply:
Fixed